### DISCUSSION TITLE
How to mark multiple defects in one drawing

### DISCUSSION THREAD

**Discussion by monkeycc** [open-edge-platform/anomalib#1374] [Feature Requests]
A picture
There are several defects at the same time

Whether the label names are all the same
All defects have one name

Or separate
If separated
How to divide data sets into folders

**Comment by sergiev** [comment]
@samet-akcay I've stumbled upon the same question in my research. Is here any support for images containing more than one kind of defect?
![image](https://user-images.githubusercontent.com/27900857/225356594-d7f988f0-a37a-4ecf-b4cd-0b265f6f135c.png)

**Comment by samet-akcay** [comment]
Currently there is no support since anomalib only labels the images as either `normal` or `abnormal`. We would like to support this in the future. However, this is also somewhat related to the literature. If there is any, can you point some papers from the literature?
